.github/workflows/deploy.yml

```diff
@@ -64,6 +64,7 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Commit to develop branch and merge to master branch
+        if: github.ref == 'refs/heads/master'
```

Comment:
I'm a bit confused with the `name` and this `if` check. The name implies that we are committing to `develop` but only if  `github.ref == 'refs/heads/master'`?